src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -624,9 +625,26 @@ $.extend($.fn.bootstrapTable.Constructor.EVENTS, {
 $.extend($.fn.bootstrapTable.defaults.icons, {
   clear: {
     bootstrap3: 'glyphicon-trash icon-clear'
-  }[$.fn.bootstrapTable.theme] || 'fa-trash'
+  }[$.fn.bootstrapTable.theme] || 'fa-trash',
+  filterControlSwitchHide: {
+    bootstrap3: 'glyphicon-zoom-out icon-zoom-out'
+  }[$.fn.bootstrapTable.theme] || 'fa-search-minus',
+  filterControlSwitchShow: {
+    bootstrap3: 'glyphicon-zoom-in icon-zoom-in'
+  }[$.fn.bootstrapTable.theme] || 'fa-search-plus'
 })
 
+$.extend($.fn.bootstrapTable.locales, {
+  formatFilterControlSwitch () {
```

Comment:
Please add the locale functions also to the locale files.

Reply:
OK, I've added them :-)
Most are just containing the English texts, for the language I know I added actual translations. Hope that is sufficient.